tests/ui/type-alias-impl-trait/no_inferrable_concrete_type.stderr

```diff
@@ -13,7 +13,7 @@ LL |     let _: foo::Foo = std::mem::transmute(0u8);
    |                       ^^^^^^^^^^^^^^^^^^^
    |
    = note: source type: `u8` (8 bits)
-   = note: target type: `Foo` (size can vary because of [type error])
+   = note: target type: `Foo` (size can vary)
```

Comment:
This message is vague. We probably should just delay as bug, instead of printing something like this.

Reply:
I've changed it delay as bug